packages/client/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
```

Comment:
Is there is reason why our lib/target/module/etc are deviating from Gutenberg: https://github.com/WordPress/gutenberg/blob/trunk/tsconfig.base.json

Reply:
I swear I aligned these 😓  (as it was already called out in one of the other PRs), but thanks for catching! The tsconfig matches Gutenberg now https://github.com/WordPress/abilities-api/pull/60/commits/4c3085ccba739c43230c40016e5c2b2f7bc4098f